TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace TelCo.ColorCoder
+{
+    class ColorPair : Colours
```

Comment:
what is the significance of Inheritance here 

ColorPair Name represents information holder but it holds construction Logic as well ?

Reply:
Modified the class names to give more absolute meaning.